### DISCUSSION TITLE
can jax with tpu be used to video to images extraction ??

### DISCUSSION THREAD

**Discussion by 1kaiser** [jax-ml/jax#7960] [Q&A] [answered]
is it possible to extract video to frames or even streaming video to frames using jax on tpu google colab??

**Comment by jakevdp** [comment, marked as answer]
There are no tools specific to video extraction built-in to JAX. You may be able to utilize JAX for some parts of the operations you have in mind, but I'm not aware of any examples of this. In particular, things like dealing with video codecs are definitely functionalities you'll have to find elsewhere.
